tests/Areas/AppConfig/UnitTests/Account/AccountListCommandTests.cs

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
```

Comment:
This is an unused import statement, please remove it :)

Reply:
You can remove them globally by using the lightbulb icon thing that appears when you hover over an unused import